docs/upgrading.rst

```diff
@@ -4,7 +4,7 @@
 Upgrade Notes
 =============
 
-See also the `CHANGELOG <../CHANGELOG>`.
+See also the :ref:`CHANGELOG <../CHANGELOG>`.
```

Comment:
It should be `:doc:` I think

Reply:
I'm not 100% that `:doc:` will work either, since `CHANGELOG` isn't included in the sphinx build? https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html?highlight=%3Adoc%3A#role-doc